gradle-plugin/src/main/kotlin/dev/icerock/gradle/generator/ImagesGenerator.kt

```diff
@@ -38,6 +39,9 @@ abstract class ImagesGenerator(
 
         val resourceClass = ClassName("dev.icerock.moko.resources", "ImageResource")
 
+        val fileResourceInterfaceClassName = ClassName("dev.icerock.moko.resources", "ResourceContainer")
+        classBuilder.addSuperinterface(fileResourceInterfaceClassName.parameterizedBy(resourceClass))
+
```

Comment:
we can add it on base - Generator. and it will be work for all generators

Reply:
in `MRGenerator.Generator`

Reply:
for support `ResourceContainer` at all generated groups

Reply:
Added `ResourceContainer` imlementation to the base `MRGenerator` class.